app/templates/google-services.json

```diff
@@ -1,52 +1,391 @@
 {
   "project_info": {
-    "project_number": "363835147397",
-    "firebase_url": "https://durable-jet-88016.firebaseio.com",
-    "project_id": "durable-jet-88016",
-    "storage_bucket": "durable-jet-88016.appspot.com"
+    "project_number": "768065119425",
+    "firebase_url": "https://commcare-a57e4.firebaseio.com",
+    "project_id": "commcare-a57e4",
+    "storage_bucket": "commcare-a57e4.appspot.com"
   },
   "client": [
     {
       "client_info": {
-        "mobilesdk_app_id": "1:363835147397:android:4aabcacd2e2d50cb",
+        "mobilesdk_app_id": "1:768065119425:android:5d470198e0a973a5",
         "android_client_info": {
-          "package_name": "richard.chard.lu.android.areamapper"
+          "package_name": "org.commcare.consumerapps.clinicalscales"
         }
       },
       "oauth_client": [
         {
-          "client_id": "363835147397-o7m4tvjb3dmnugun511gcjohqae4isof.apps.googleusercontent.com",
-          "client_type": 1,
-          "android_info": {
-            "package_name": "richard.chard.lu.android.areamapper",
-            "certificate_hash": "b170b8c2d0571d502c869cf81862d7be1019db26"
-          }
+          "client_id": "768065119425-hg10t0osvjqrt6pne3toaobabn5gobqs.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyDDh8PWdeUSvv55vUQsRIO_t23GUukqJAw"
+        },
+        {
+          "current_key": "AIzaSyDpLXffDgRluouKQ54Z4hSPqRORBbTwjcY"
         },
         {
-          "client_id": "363835147397-t85mfsavpsogn159gpohiekvc18jb1m2.apps.googleusercontent.com",
+          "current_key": "AIzaSyDx6neJf_c4NA_17mNZ9e_aD4-SGJyig8c"
+        },
+        {
+          "current_key": "AIzaSyC29AadllbS3sZ3Jq_72Ae-wQaN8ykEx3g"
+        }
+      ],
+      "services": {
+        "appinvite_service": {
+          "other_platform_oauth_client": [
+            {
+              "client_id": "768065119425-05thotts3f4ujdcscoaeheebpia8ag9m.apps.googleusercontent.com",
+              "client_type": 3
+            }
+          ]
+        }
+      },
+      "admob_app_id": "ca-app-pub-8038725004530429~5040587593"
+    },
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:768065119425:android:455113c70d7e361f",
+        "android_client_info": {
+          "package_name": "org.commcare.consumerapps.germygameshindi"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "768065119425-hg10t0osvjqrt6pne3toaobabn5gobqs.apps.googleusercontent.com",
           "client_type": 3
         }
       ],
       "api_key": [
         {
-          "current_key": "@apiKey@"
+          "current_key": "AIzaSyDDh8PWdeUSvv55vUQsRIO_t23GUukqJAw"
```

Comment:
we should not commit the api key here publicly but keep it as `@apiKey@`. Jenkins inject this key at build time into this file.

Reply:
you also don't need to define multiple keys here. Look at https://github.com/dimagi/commcare-android/blob/master/app/templates/google-services.json as an example.

Reply:
Yeah, I missed that, let me correct it.